src/widget/wtrackmenu.cpp

```diff
@@ -169,6 +169,10 @@ void WTrackMenu::createActions() {
         m_pImportMetadataFromMusicBrainzAct = new QAction(tr("Import From MusicBrainz"), m_pMetadataMenu);
         connect(m_pImportMetadataFromMusicBrainzAct, &QAction::triggered, this, &WTrackMenu::slotShowDlgTagFetcher);
 
+        // Give a NULL parent because otherwise it inherits our style which can
```

Comment:
```suggestion
        // Give a nullptr parent because otherwise it inherits our style which can
```
`NULL` is a hack we used before C++11 introduced `nullptr`